orderbook/openapi.yml

```diff
@@ -153,8 +153,34 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/FeeInformation"
-        400:
+        404:
           description: sellToken not existent
+  /api/v1/trades/:
+    get:
+      summary: Get existing Trades.
+      description: |
+        By default all trades are returned.
+        Queries can be refined by specifiying owner or order_uid.
+      parameters:
+        - name: owner
+          in: query
+          schema:
+            $ref: "#/components/schemas/Address"
+          required: false
+        - name: order_uid
+          in: query
+          schema:
+            $ref: "#/components/schemas/UID"
+          required: false
+      responses:
+        200:
+          description: all trades
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Trade"
+        404:
+          description: No trades found.
```

Comment:
I think the return type should be array of Trade like for the orders route and we don't need 404 because if there are no trades then the result is an empty array.